plugin/pkg/admission/limitranger/admission.go

```diff
@@ -249,6 +249,42 @@ func mergePodResourceRequirements(pod *api.Pod, defaultRequirements *api.Resourc
 		}
 	}
 
+	for i := range pod.Spec.InitContainers {
+		container := &pod.Spec.InitContainers[i]
```

Comment:
can we factor this to just have a mergeContainer function so its shared with init and non-init containers?